docs/users/developer-tools.md

```diff
@@ -23,6 +23,18 @@ http://mysite.ddev.local:8025
 
 Please note this will not intercept emails if your application is configured to use SMTP or a 3rd-party ESP integration. If you are using SMTP for outgoing mail handling ([Swiftmailer](https://www.drupal.org/project/swiftmailer) or [SMTP](https://www.drupal.org/project/smtp) modules for example), update your application configuration to use `localhost` on port `1025` as the SMTP server locally in order to use MailHog.
 
+### Database Management
+
+[phpMyAdmin](https://www.phpmyadmin.net/) is a free software tool to manage MySQL and MariaDB databases from a browser. phpMyAdmin comes installed with ddev. 
+
+After your project is started, access the phpMyAdmin web interface at its default port:
+
+```
+http://mysite.ddev.local:8036
+```
+
+If you use [Sequel Pro](https://www.sequelpro.com/) for macOS, run `ddev sequelpro` within a project folder, and Sequel Pro will launch and load the database for that project. Sequel Pro is a free open source database browser for macOS. 
```

Comment:
Maybe "access the database" would be better than "load the database"? Certainly it's not loading a db.  

Probably better to just say "If you use the free Sequel Pro..." and drop the last sentence.

Reply:
k! sorted.